### PR TITLE
Fix/cli help language

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Updates the language in some CLI help messages.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/agents-aea/blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/packages.py [hunk 1]
```diff
@@ -66,15 +66,15 @@ def package_manager(
 @click.option(
     "--update-packages",
     is_flag=True,
-    help="Update packages if the calculated hash for a package does not match the one in the packages.json.",
+    help="Download packages from a remote registry so local packages match the hashes in packages.json.",
```
Comment (jmoreira-valory) [inline aea/cli/packages.py:69]: "so that local packages..."

File: aea/cli/generate_all_protocols.py [hunk 2]
```diff
@@ -445,7 +445,7 @@ def _process_packages_protocol(
 @click.option(
     "--check-clean",
     is_flag=True,
-    help="Check if git tree is clean..",
+    help="Check if git tree is clean.",
```
Comment (jmoreira-valory) [inline aea/cli/generate_all_protocols.py:448]: Git

File: aea/cli/packages.py [hunk 3]
```diff
@@ -104,7 +104,7 @@ def sync(ctx: Context, update_packages: bool, update_hashes: bool) -> None:
 )
 @pass_ctx
 def lock_packages(ctx: Context, check: bool) -> None:
-    """Lock packages"""
+    """Lock packages. Updates hashes in packages.json so they match the local packages."""
```
Comment (jmoreira-valory) [inline aea/cli/packages.py:107]: Update hashes

so that they match

File: docs/api/plugins/aea_cli_benchmark/case_acn_communication/command.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ Check amount of time for acn connection start.
 @click.option(
     "--connect-times",
     default=10,
-    help="How many time perform connection.",
+    help="How many times to perform connection.",
```
Comment (jmoreira-valory) [inline docs/api/plugins/aea_cli_benchmark/case_acn_communication/command.md:23]: How many connection retries.
How many connection attempts.
(?)

File: docs/api/plugins/aea_cli_benchmark/case_acn_startup/command.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ Check amount of time for acn connection start.
 @click.option(
     "--connect-times",
     default=10,
-    help="How many time perform connection.",
+    help="How many times to perform connection.",
```
Comment (jmoreira-valory) [inline docs/api/plugins/aea_cli_benchmark/case_acn_startup/command.md:23]: Same as above.

File: plugins/aea-cli-benchmark/aea_cli_benchmark/case_acn_startup/command.py [hunk 6]
```diff
@@ -50,7 +50,7 @@
 @click.option(
     "--connect-times",
     default=10,
-    help="How many time perform connection.",
+    help="How many times to perform connection.",
```
Comment (jmoreira-valory) [inline plugins/aea-cli-benchmark/aea_cli_benchmark/case_acn_startup/command.py:53]: Same as above.

**Review by jmoreira-valory — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**